src/Playwright/API/Generated/IFrame.cs

```diff
@@ -636,8 +650,7 @@ public partial interface IFrame
         /// selector, the first will be used. See <a href="./selectors.md">working with selectors</a>
         /// for more details.
         /// </param>
-        /// <param name="options">Call options</param>
-        Task<bool> IsHiddenAsync(string selector, FrameIsHiddenOptions? options = default);
+        Task<bool> IsHiddenAsync(string selector);
```

Comment:
This is an API breaking change, we should probably revert it upstream for simplicity.

Reply:
https://github.com/microsoft/playwright/pull/7600